deployment/ansible/roles/nyc-trees.app/templates/nginx-nyc-trees-app.conf.j2

```diff
@@ -6,6 +6,20 @@ server {
 
   access_log /var/log/nginx/nyc-trees-app.access.log logstash_json;
 
+  location /favicon.ico {
```

Comment:
I'm not sure this will work, because in this case the mentioned `favicon.ico` file will have a hash segment appended to it in production. i just deleted my static folder and ran `npm run build` and this is my output:
```
vagrant@app:/var/www/nyc-trees/static/img/favicon$ ls
android-chrome.3383adc6.png    favicon.cabfcaf1.ico
apple-touch-icon.1d8bb3b9.png  manifest.f18abea0.json
```
So I think the easiest way to handle it is to make a copy in a separate path that is outside of the rewrite rules, as I've done with this PR. Does that sound right?